test/functional/apps/console/monaco/_misc_console_behavior.ts

```diff
@@ -132,14 +132,20 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     describe('customizable font size', () => {
       // flaky
```

Comment:
I actually meant the previous test (on line 117) 😅 This one appears to be flaky, hopefully with the new changes it's not anymore - could we run the flaky test runner to verify (and remove the `// flaky` comment if fixed)?

Reply:
Flaky test runner x100 all green https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/6818. Gonna merge this in!